.goreleaser.yml

```diff
@@ -41,3 +41,18 @@ release:
   github:
     owner: sonatype-nexus-community
     name: nancy
+
+dockers:
+  - 
+    goos: linux
+    goarch: amd64
+    dockerfile: Dockerfile
+    build_flag_templates:
+    - "--label=author='DJ Schleen'"
+    - "--label=version={{ .Version }}"
+    - "--label=maintainer='sonatype-nexus-community'"
+    image_templates:
+    - "sonatypecommunity/nancy:latest"
+    - "sonatypecommunity/nancy:{{ .Tag }}"
+    - "sonatypecommunity/nancy:v{{ .Major }}"
+    - "sonatypecommunity/nancy:v{{ .Major }}.{{ .Minor }}"
```

Comment:
I need to get up to speed on the goreleaser doco, but at first glance, the Tag/Major/Minor smells funny. My in brain semver parser expects the last two fields to be Minor/Patch, e.g. Major/Minor/Patch.

Reply:
This is a Docker thing, more or less so you can say "I don't mind getting updates on a patch level" or minor level, etc...

Reply:
It's essentially like "I want to be less dangerous than using latest, but I'll live a little bit dangerously".